ingestion-beam/src/main/java/com/mozilla/telemetry/util/Json.java

```diff
@@ -204,17 +205,12 @@ public static String asString(Object data) throws IOException {
    * {@code getAttributeMap} method returns the value for the {@code attributes} parameter.
    * Additionally jackson doesn't like that there are no setter methods on {@link PubsubMessage}.
    *
-   * <p>Beam 2.16 added the {@code messageId} field, which is not relevant for our purposes;
```

Comment:
I think we should still have some commentary here about why we mark the field as `@JsonIgnore`.

Am I correct that with the changes here, we are able to read messageId when reading JSON input, but we won't output messageId when serializing to JSON?

Reply:
Yes, that is correct